Makefile

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
I'm not sure what value this make file provides?
It only runs powershell scripts and introduces another dependency in the toolchain.

Reply:
Open to explaining my reasons 🙂 :

- it standardizes over using commands together. e.g. : you only need to type `make it description=${FILE} language=${LANGUAGE}` instead of having to dig down in the CI which commands to type and the scripts and programs to execute
- the execution of scripts and applications it's documented and is easier to remember/build upon
- improves the "first contribution experience", especially for "non-dotnet" developers
- it's a pretty common and familiar way to get started with an open source project

This said, I'm not too fussy, we can achieve "almost" the same goal with additional scripts wrapping the pre-existing, but we would need to have a little separate documentation to explain the usage.

Reply:
Thanks for providing context here.
Here are my concerns:
- make is not standard tooling for most windows devs (except for the ones doing C/C++/ASM maybe)
- I'm not sure the call to **shell** has any meaning outside of make/linux

I'd be supportive of using/improving the PowerShell scripts we already have. What do you think?

Reply:
Out of curiosity, have you tried the current `Makefile`  on Win? As far as I understand it, [`shell`](https://www.gnu.org/software/make/manual/html_node/Shell-Function.html) should work out of the box.

Anyhow, let's agree on iterating over the current PowerShell scripts and removing the dependency on `Make`.
I move it to Draft meanwhile.

Reply:
@baywet I removed the `Makefile` and separated the PowerShell commands, let me know if this is a direction we can agree on 🙂